scripts/setup.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+# ASCII art
+echo "  _           _           _             "
+echo " | |         | |         | |            "
+echo " | | ___   __| | ___  ___| |_ __ _ _ __ "
+echo " | |/ _ \ / _  |/ _ \/ __| __/ _  |  __|"
+echo " | | (_) | (_| |  __/\__ \ || (_| | |   "
+echo " |_|\___/ \__ _|\___||___/\__\__ _|_|   "
+echo ""
+
+# Declare a log file to capture detailed logs and a temp directory
+LOGFILE="$HOME/lodestar-app-temp/logfile.log"
+TEMP_DIR="$HOME/lodestar-app-temp"
```

Comment:
this isn't really a temporary directory as it's used to store the unpacked binary, might be better to use `/tmp/` to do the download and unpacking, the just move the binary to `~/.local/bin`, we could give the user prompt at the beginning of the script to choose a custom location to store the binary in, but it should act similar to if you just `apt install` which most of the time copies it to global or user specific `/bin` folder

Reply:
> we could give the user prompt at the beginning of the script to choose a custom location to store the binary in

on second thought, probably don't even need that, we can just print out where the binary is located and if they want they can move it afterwards

Reply:
I like the `mktemp` usage :+1: